src/utils/mock_data/pullData.js

```diff
@@ -0,0 +1,98 @@
+const { Octokit } = require('@octokit/core');
+const path = require('path');
+const dotenv = require('dotenv');
+const usersInfos = require('./users');
+
+// range: [min, max)
+function getRandomInt(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min) + min);
+}
+  
+
+dotenv.config({ path: path.resolve(__dirname, '../../../.env') });
+
+function repoFormat(repo, id, creator) {
+    let project = {};
+    project.id = id;
+    project.name = repo.name.replace(new RegExp('-', 'g'), ' ');
+    project.creator = creator;
+    project.description = repo.description;
+    project.skills = repo.languages;
+    project.tags = repo.topics.map(topic => topic.replace(new RegExp('-', 'g'),' '));
+    project.links = [{ type: 'GitHub', link: repo.html_url }]
+
+    return project;
+}
+
+
+/**
+ * @param {string} query - search query
```

Comment:
Where'd you learn how to document like this?

Reply:
VSCode has built in support for [JSDoc](https://jsdoc.app/index.html) which I thought would be useful here because documenting with it allows you to see the specified arguments and return type when you hover your cursor over the function, so you don't have to keep scrolling up and down to double check.

Reply:
That's amazing